spec/api/comment_thread_spec.rb

```diff
@@ -477,6 +477,18 @@ def test_unicode_data(text)
         check_thread_result_json(nil, thread, parsed)
       end
 
+      context 'when requesting the thread for informational purposes' do
+        subject do
+          get "/api/v1/threads/#{thread.id}", with_responses: false # we're asking for no responses here.
```

Comment:
So, after thinking about this a lot, here are my thoughts:

#1 - Do the work described above for the LMS/Discussion API: make them explicitly pass true/false for getting responses.  I think that the `common_client`/`django_comment_client` code is already convoluted enough that we could stand to make it declarative by forcing the selection of responses/no responses there.  Less to understand about an ancillary service.

#2 - Release the LMS changes, as described above.  They'll be sending `with_responses` as true or false, but the comment service won't be honoring it yet.  Status quo is preserved.

#3 - Leave this PR as-is, and actually have the default behavior, in lieu of a `with_responses` query parameter, be to include the responses.  This preserves the existing behavior for consumers of the service that aren't the LMS, and since the LMS is already being explicit, we're still getting the benefits.

As nice as it would be to have things not returned by default from the service, I think that approach is more heavy handed and doesn't fit the spirit of the service; we would be bending the comment service endpoints around lackluster caller code.  This way, we're simply adding more flexibility to let the caller make intelligent requests.